crates/rnote-compose/src/splitorder.rs

```diff
@@ -12,20 +12,33 @@ use serde::{Deserialize, Serialize};
     Deserialize,
     num_derive::FromPrimitive,
     num_derive::ToPrimitive,
+    Default,
 )]
+#[cfg_attr(feature = "clap-derive", derive(clap::ValueEnum))]
 #[serde(rename = "split_order")]
 pub enum SplitOrder {
+    #[default]
     /// Split in row-major order.
     #[serde(rename = "row_major")]
+    #[cfg_attr(feature = "clap-derive", value(name = "horizontal"))]
```

Comment:
Rename this to `horizontal-first`, `vertical-first` to match the export option in the GUI